server/internal/app/server.go

```
@@ -69,22 +72,29 @@
 
 func (w *WebServer) Run() {
-	defer log.Infof("Server shutdown")
-
 	debugLog := ""
 	if w.appServer.Debug {
 		debugLog += " with debug mode"
 	}
 	log.Infof("server started%s at http://%s\n", debugLog, w.address)
 
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, os.Interrupt, syscall.SIGTERM, os.Kill)
```

golangci-lint (1.64.8)

SA1016: os.Kill cannot be trapped (did you mean syscall.SIGTERM?)

(staticcheck)